ci-operator/step-registry/ibmcloud/deprovision/vpc/ibmcloud-deprovision-vpc-commands.sh

```diff
@@ -101,6 +101,48 @@ echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpc_name
 echo "DEBUG" "Removing the vpc ${vpc_name} ..."
 delete_vpc "${vpc_name}"
 
+hasError=0
+echo "DEBUG" "Checking the remains resources and the last_operation of the resources in ${resource_group}..."
```

Comment:
Suggest to add a new step for your check purpose to avoid the change on the vpc deprovision step.

Reply:
how about adding step ci-operator/step-registry/cucushift/installer/check/ibmcloud/destroy/cucushift-installer-check-ibmcloud-destroy-commands.sh

Reply:
okay for me.